python/lsst/sims/catUtils/exampleCatalogDefinitions/phoSimCatalogExamples.py

```diff
@@ -290,15 +301,30 @@ class PhoSimCatalogZPoint(PhosimInputBase, PhoSimAstrometryGalaxies, EBVmixin):
     transformations = {'raPhoSim': np.degrees, 'decPhoSim': np.degrees}
 
 
+class PhoSimCatalogSN(PhoSimCatalogZPoint, FrozenSNCat, EBVmixin):
+    catalog_type = 'phoSim_SNcatalog'
+
+    def get_sedFilepath(self):
+        return self.column_by_name('TsedFilepath')
+
+    def get_phoSimMagNorm(self):
+        return self.column_by_name('TmagNorm')
+
+    default_formats = {'S':'%s', 'f':'%.9g', 'i':'%i'}
```

Comment:
Since you inherit from `PhoSimCatalogZPoint`, you do not need to re-define `default_formats`, `spatial_model`, or `transformations`.